### PR TITLE
RPM specfile cleanup

### LINKED ISSUE #265: RPM specfile cleanup
This issue is just for documenting the job already done by @xavierba in MR #246 

The goal is to clean the specfile with:
- typos
- good practices
- obsolete techniques

### PR DESCRIPTION
**Pull request by xavierba**
Extensive RPM specfile cleanup.

Closes #265

### CODE REVIEW

File: src/install/lsc-core.spec [hunk 1]
```diff
@@ -103,121 +94,130 @@ install -p -m 0644 lib/systemd/system/lsc-async@.service %{buildroot}%{_unitdir}
...
+%license LICENSE.txt
+%doc README.md doc/html/
+%doc sample/ etc/lsc.xml-sample etc/sql-map-config.xml-sample
+%doc etc/sql-map-config.d/InetOrgPerson.xml-sample
+%dir %{_sysconfdir}/lsc/
+%config(noreplace) %{_sysconfdir}/lsc/*.xml
+%dir %{_sysconfdir}/lsc/sql-map-config.d/
+%config(noreplace) %{_sysconfdir}/lsc/sql-map-config.d/InetOrgPerson.xml
+%config(noreplace) %{_sysconfdir}/cron.d/lsc
+%config(noreplace) %{_sysconfdir}/default/lsc
```
Comment (davidcoutadeur) [inline src/install/lsc-core.spec:156]: I think some lines have been dropped during the rebase.

Here we should:

1. remove: (this file does not exist anymore)

```
%config(noreplace) %{_sysconfdir}/default/lsc
```

2. add:

```
%config(noreplace) %{_sysconfdir}/default/lsc-sync
%config(noreplace) %{_sysconfdir}/default/lsc-async
```

All other modification seems, ok. Thanks for you hard work!

Comment (xavierba) [reply src/install/lsc-core.spec]: good catch, fixed.

**Comment by davidcoutadeur** [comment]
I have done a complete review and testing of this issue. For me, it can merged.

**Comment by coudot** [comment]
> I have done a complete review and testing of this issue. For me, it can merged.

Feel free to merge it, it's ok for me too.

**Pull request merged**